src/app/CoordinateSectorCreator.js

```diff
@@ -0,0 +1,382 @@
+define([
+    'dijit/_TemplatedMixin',
+    'dijit/_WidgetBase',
+
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+    'dojo/_base/array',
+    'dojo/_base/event',
+
+    'dojo/query',
+    'dojo/number',
+    'dojo/on',
+    'dojo/aspect',
+    'dojo/dom-attr',
+    'dojo/dom-class',
+    'dojo/text!app/templates/CoordinateSectorCreator.html',
+
+    'esri/geometry/Point',
+    'esri/SpatialReference',
+    'esri/geometry/webMercatorUtils',
+    'esri/geometry/Polygon',
+    'esri/geometry/Circle',
+    'esri/symbols/SimpleFillSymbol',
+    'esri/graphic',
+    'esri/layers/GraphicsLayer'
+], function (
+    _TemplatedMixin,
+    _WidgetBase,
+
+    declare,
+    lang,
+    array,
+    events,
+
+    query,
+    number,
+    on,
+    aspect,
+    domAttr,
+    domClass,
+    template,
+
+    Point,
+    SpatialReference,
+    webMercatorUtils,
+    Polygon,
+    Circle,
+    SimpleFillSymbol,
+    Graphic,
+    GraphicsLayer
+) {
+    return declare([_WidgetBase, _TemplatedMixin], {
+        // description:
+        //      Creates circle sectors from coordinates.
+        templateString: template,
+        baseClass: 'coordinate-sector-creator',
+        // _panelController: {key: DomNode}
+        // summary:
+        //      an object has of domnodes that contain the
+        //      coorindate form elements
+        _panelController: null,
+        // zoomLevel: Number
+        // summary:
+        //      the cache level to zoom the map to
+        zoomLevel: 13,
+
+        // Properties to be sent into constructor
+        // map: esri/map
+        // summary:
+        //      the map to zoom
+        map: null,
+        symbol: null,
+        graphicsLayer: null,
+
+        postCreate: function () {
+            // summary:
+            //      Overrides method of same name in dijit._Widget.
+            console.log('app.CoordinateSectorCreator::postCreate', arguments);
+            if (!this.map) {
+                throw 'This widget requires an esri/map to be useful.';
+            }
+            this.symbol = new SimpleFillSymbol().setColor(null).outline.setColor('red');
```

Comment:
Does this yellow and thicker lines look better?
![linevis](https://cloud.githubusercontent.com/assets/5684657/15371865/cf915608-1cfa-11e6-800d-dcb567dbec47.jpg)